Xamarin.Forms.Platform.WPF/Controls/FormsProgressRing.xaml.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media.Animation;
+
+namespace Xamarin.Forms.Platform.WPF.Controls
+{
+	public partial class FormsProgressRing : UserControl
+	{
+		public static readonly DependencyProperty IsActiveProperty = DependencyProperty.Register("IsActive", typeof(bool), typeof(FormsProgressRing), new PropertyMetadata(false, new PropertyChangedCallback(IsActiveChanged)));
+
+		Storyboard animation;
+
+		public FormsProgressRing()
+		{
+			InitializeComponent();
+
+			animation = (Storyboard)Resources["ProgressRingStoryboard"];
+		}
+
+		public bool IsActive
+		{
+			get
+			{
+				return (bool)GetValue(IsActiveProperty);
+			}
+
+			set
+			{
+				SetValue(IsActiveProperty, value);
+			}
+		}
+
+		private static void IsActiveChanged(DependencyObject sender, DependencyPropertyChangedEventArgs e)
```

Comment:
let's drop the `private` on these methods

Reply:
Fixed this now

Reply:
Thanks, building now, I m going to test this.